var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -47,14 +48,18 @@ class Cp2k(Package):
     depends_on('lapack')
     depends_on('blas')
     depends_on('fftw')
-    depends_on('libint@:1.2', when='@3.0')
+    depends_on('libint@:1.2', when='@3.0,4.1')
 
     depends_on('mpi', when='+mpi')
     depends_on('scalapack', when='+mpi')
     depends_on('plumed+shared+mpi', when='+plumed+mpi')
     depends_on('plumed+shared~mpi', when='+plumed~mpi')
-    depends_on('pexsi', when='+mpi')
-    depends_on('wannier90', when='+mpi')
+    depends_on('pexsi+fortran', when='+mpi')
+
+    # Apparently cp2k@4.1 needs an "experimental" version of libwannier.a
+    # which is only available contacting the developer directly. See INSTALL
+    # in the stage of cp2k@4.1
```

Comment:
Tell me a bit more about this. Does this mean that cp2k@4.1 can't be built with Spack? If so we should make the older version the default.